Formula/grpc.rb

```diff
@@ -0,0 +1,44 @@
+class Grpc < Formula
+  desc "Google's open-source RPC framework, based on protocol buffers."
+  homepage "http://www.grpc.io/"
+  head "https://github.com/grpc/grpc.git"
+
+  stable do
```

Comment:
I don't think a `stable` block is necessary here, this can just be collapsed into the formula body.